src/extensions/http-load-balancer.ts

```diff
@@ -72,7 +72,7 @@ export class HttpLoadBalancerExtension extends ServiceExtension {
       }
       this.parentService.scalableTaskCount.scaleOnRequestCount(`${this.parentService.id}-target-request-count-${this.requestsPerTarget}`, {
         requestsPerTarget: this.requestsPerTarget,
-        targetGroup,
+        targetGroup: this.parentService.targetGroup || targetGroup,
```

Comment:
this will always result in this line using `targetGroup`, not `this.parentService.targetGroup`.

parentService.targetGroup isn't set. Please add a line somewhere which sets the target group of the parentService so that customers can actually use it.